app/screens/transactions/Transactions.js

```diff
@@ -2,171 +2,199 @@
-              <Button text="Received" onClick={() => this.setState({ isReceivedDisplayed: !isReceivedDisplayed })} isActive={isReceivedDisplayed} style={btnStyle} />
-              <Button text="Pending" onClick={() => this.setState({ isPendingDisplayed: !isPendingDisplayed })} isActive={isPendingDisplayed} style={btnStyle} />
-              <Button text="Rejected" onClick={() => this.setState({ isRejectedDisplayed: !isRejectedDisplayed })} isActive={isRejectedDisplayed} style={btnStyle} />
-            </ButtonsWrapper>
-            <TransactionsListWrapper>
-              {transactions ? (
-                transactions.map((tx, index) => (
-                  <Transaction
-                    key={index}
-                    transaction={tx}
-                    isSentDisplayed={isSentDisplayed}
-                    isReceivedDisplayed={isReceivedDisplayed}
-                    isPendingDisplayed={isPendingDisplayed}
-                    isRejectedDisplayed={isRejectedDisplayed}
-                  />
-                ))
-              ) : (
-                <RightSectionText>No transactions executed yet</RightSectionText>
-              )}
-            </TransactionsListWrapper>
-          </LeftSection>
-          <RightSection>
-            <RightSectionSubHeader>Crypto Transactions</RightSectionSubHeader>
-            <RightSectionText>
-              Lorem ipsum dolor sit amet, consetetur sadipscing elitr, sed diam nonumy eirmod tempor invidunt ut labore et dolore magna aliquyam erat, sed diam voluptua. At vero
-              eos et accusam et justo duo dolores et ea.
-            </RightSectionText>
-            <RightSectionLink onClick={this.navigateToExplanation}>Learn more about Spacemesh Transactions</RightSectionLink>
-          </RightSection>
-        </InnerWrapper>
+    const { history } = this.props;
+    const { selectedItemIndex, filteredTransactions, mined, sent, received, totalMined, totalSent, totalReceived, addressToAdd } = this.state;
+    return (
+      <Wrapper>
+        <SecondaryButton onClick={history.goBack} img={chevronLeftWhite} imgWidth={7} imgHeight={10} style={{ position: 'absolute', left: -35, bottom: 0 }} />
+        <WrapperWith2SideBars width={680} height={480} header="TRANSACTION LOG" style={{ marginRight: 10 }}>
+          <BoldText>Latest transactions</BoldText>
+          <Link onClick={this.navigateToContacts} text="MY CONTACTS" style={{ position: 'absolute', top: 30, left: 550 }} />
+          <TransactionsListWrapper>
+            {filteredTransactions && filteredTransactions.length ? (
+              filteredTransactions.map((tx, index) => (
+                <TransactionRow key={index} transaction={tx} addAddressToContacts={({ address }) => this.setState({ addressToAdd: address })} />
+              ))
+            ) : (
+              <Text>No transactions executed yet</Text>
+            )}
+          </TransactionsListWrapper>
+        </WrapperWith2SideBars>
+        <RightPaneWrapper>
```

Comment:
wrap DropDown and TransactionsMeta in single wrapper RightPaneWrapper and change between it to CreateNewContact so the UI will look much better